File: app/src/main/java/in/techrebounce/todonotes/model/Notes.java
package in.techrebounce.todonotes.model;

public class Notes {
    private String title;
    private String description;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "Notes{" +
                "title='" + title + '\'' +
                ", description='" + description + '\'' +
                '}';
    }
}

File: app/src/main/java/in/techrebounce/todonotes/db/Notes.kt
package `in`.techrebounce.todonotes.db

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

// structure of table
@Entity(tableName = "notesData")
data class Notes (
        @PrimaryKey(autoGenerate = true)
        var id : Int? = null,
        @ColumnInfo(name = "title")
        var title : String = "",
        @ColumnInfo(name = "description")
        var description : String = "",
        @ColumnInfo(name = "imagePath")
        var imagePath : String = "",
        @ColumnInfo(name = "isTaskCompleted")
        var isTaskCompleted : Boolean = false
)